Add JUnit config to intellijinit

pejovica opened the pull request:
Makes intellijinit generate default run/debug configuration for JUnit.

There are two aspects to proposed changes. First one is the placement of generated configuration. At the moment it is stored in the workspace.xml, since the official location is not available yet (IDEA-65915). And the second one is the collection of necessary VM parameters for JUnit to run. I tried to reuse mx unittest for that by effectively making it execute a dry run. This implementation of dry run only prevents tests from running. It doesn't prevent side effects, e.g., unittest's test discovery and caching. But since unittest allows custom launchers, i.e., implemented by other modules, I am not sure what can be expected from them in this regard. Graal core seems to work fine.

pejovica committed:
Add JUnit config to intellijinit

vjovanov commented:
@pejovica could you please sign the oracle contributor agreement? Here are the instructions: 
  http://www.oracle.com/technetwork/community/oca-486395.html
Also, the gate is failing due to style issues. To reproduce locally run: 
```
mx --strict-compliance gate --strict-mode
```

vjovanov commented:
Otherwise, LGTM

pejovica committed:
Fix pylint complaints

pejovica committed:
Add some tweaks to IntelliJ support

 - Make IntelliJ builds fail on Ant Exec task error

 - Pass IntelliJ project's JDK to Ant build tasks, so that they are
   independent of the environment (i.e., they run even without
   JAVA_HOME being set)

 - Make ideclean remove IntelliJ's modules that were created by
   running intellijinit with --mx-python-modules flag

pejovica committed:
Generate minimum configuration without template

pejovica commented:
I've added the final set of changes, so could you have another look?

vjovanov commented:
@pejovica will there be progress on this or we can close the PR?

vjovanov commented:
@pejovica is this still valid?
